### PR TITLE
xApi for learning path template.

### PR DESCRIPTION
**Pull request by vasyl-ivanchuk**
main changes:
* xApi implementation including Login and Error views.
* progress save/reset functionality including exit button and dropdown with options to save/reset progress.
* finish course popup.
* autologin for courses.

### CODE REVIEW

File: app/dialogs/finishCourse.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+﻿<section class="popup-layout">
```
Comment (LeonidPelyhivskyi) [inline app/dialogs/finishCourse.html]: @vasyl-ivanchuk  maybe better to name the file as "finishLearningPath.html"

File: app/dialogs/finishCourse.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿define([], function () {
```
Comment (LeonidPelyhivskyi) [inline app/dialogs/finishCourse.js]: @vasyl-ivanchuk this too

**Comment by vasyl-ivanchuk** [comment]
Renamed exitCourse to exitPath, finishCourse to finishPath. I've decided to use Path instead of LearningPath, because of shorter naming. On button labels we use Path but not Learning Path as well.

**Comment by LeonidPelyhivskyi** [comment]
Good job!;) Reviewed.

**Pull request merged**